### DISCUSSION TITLE
CUYANIMELIST | sesi 1 pemindahan folder

### DISCUSSION THREAD

**Discussion by Destaanugra1** [deaafrizal/forum#42] [Troubleshooting Room] [answered]
![Screenshot (120)](https://github.com/deaaprizal/forum/assets/152362800/8df65687-a613-4ea6-ba55-041fb2371c33)
![Screenshot (119)](https://github.com/deaaprizal/forum/assets/152362800/757ccf90-dd7e-43ce-87d2-109440d27d80)
keapa errro di situ yang saat mindahin foldernya ke src padalah sesuai buat navbarnya saran dong bang

**Comment by andikaalakate** [comment, marked as answer]
Di bagian import Navbar dalam file layout.js tuh seharusnya Utilities, bukan Utilites, coba lihat lagi import locationnya, bang...

**Reply by Destaanugra1** [reply]
thaks bang

**Reply by andikaalakate** [reply]
Syappp~
